icu4c/source/common/unicode/umutablecptrie.h

```diff
@@ -235,4 +216,23 @@ umutablecptrie_buildImmutable(UMutableCPTrie *trie, UCPTrieType type, UCPTrieVal
 
 U_CDECL_END
 
+#if U_SHOW_CPLUSPLUS_API
```

Comment:
Wasn't this change in some other PR.

Reply:
I need to update my trunk

Reply:
https://github.com/unicode-org/icu/pull/1095 is landed into maint67 and not yet merged back to the trunk yet. This PR is diff from the trunk and I need the change in that PR to compile/link correctly. Once 1095 merge back to the trunk we should see this part resolve.